Graphs/Table/index.js

```diff
@@ -419,22 +420,36 @@ const TableGraph = (props) => {
         // top calculate position of tooltip, -20 is to show tooltip below element when element at top of table otherwise -15 is to show tooltip above element.
         const top = rowIndex === currentStartIndex ? (rowHeight * (rowIndex + 1)) - 20 : (rowHeight * rowIndex) - 15;
 
+        //calculate tooltip left position based on the column width. column width default is 150
+        let left = 0;
+        if (cellData){
+            for(var ii=0; ii < columnIndex ; ii++) {
+                if (columns[ii].width) {
+                    left = left + (parseInt(columns[ii].width))
+                } else {
+                    left = left + 150;
+                }
+            }
+            left = left - 3 * cellData.length
```

Comment:
Can you please add a code comment explaining why you are subtracting `3` from `left` here.

Reply:
This is from the eixisting code:
            overridePosition={() => ({ left: (graphWidth / columns.length) * columnIndex - 3 * cellData.length, top })}
Here left position is calculated this way. If I dont include this then the tooltip is going out of sight. So I am just reusing the exiting code's part of "3 * cellData.length". I am only changing the 1st part - (graphWidth / columns.length) * columnIndex. Here the column width is assumed to be equal and it is multiplied by the column of columns before this tooltip index. I changed that part to add sum individual column widths before this tooltip index